frontend/app/configuration/Config.scala

```diff
@@ -74,12 +74,12 @@ object Config {
   val googleAnalyticsTrackingId = config.getString("google.analytics.tracking.id")
 
   val facebookJoinerConversionTrackingId =
-    Tier.all.map { tier => tier -> config.getString(s"facebook.joiner.conversion.${tier.slug}") }.toMap
+    Tier.allPublic.map { tier => tier -> config.getString(s"facebook.joiner.conversion.${tier.slug}") }.toMap
```

Comment:
I think we should deprecate generating configuration keys programatically in the future
being able to reliably find usages of configuration keys is convenient